tool/tctl/common/resource_command.go

```diff
@@ -3837,7 +3837,8 @@ func checkCreateResourceWithOrigin(storedRes types.ResourceWithOrigin, resDesc s
 func checkUpdateResourceWithOrigin(storedRes types.ResourceWithOrigin, resDesc string, confirm bool) error {
 	managedByStatic := storedRes.Origin() == types.OriginConfigFile
 	if managedByStatic && !confirm {
-		return trace.BadParameter(`The %s resource is managed by static configuration. We recommend removing configuration from teleport.yaml, restarting the servers and trying this command again.
+		return trace.BadParameter(`The %s resource is managed by static configuration in your Auth Service teleport.yaml.
+You can either statically set the authentication configuration there, or unset the "auth_service.authentication" field and dynamically manage the config via the "cluster_auth_preference" resource.
```

Comment:
I don't think we have to mention the ability to dynamically manage the configuration via the resource if we're getting this error message because the user wanted to dynamically manage the configuration.

Reply:
I thin we should explain the 2 options because we made bad product choices and told people "just edit cap" in the docs while cap is unusable in most self-hosted deployments. Users might come thinking "the docs told me do to X so I must use dynamic". Showing the 2 options makes it clearer that this is an "OR".